### PR TITLE
add verify v0.0.1 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/jpillora/verifyjs
**Watch 5
  Star 118
 Fork 50**
@Piicksarn could u help me check this PR for #6268 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/verify/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "verify",
+  "version": "0.0.1",
+  "description": "A powerful asynchronous validation library",
+  "filename": "verify.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jpillora/verifyjs.git"
+  },
+  "keywords": [
```
Comment (PeterDaveHello) [inline ajax/libs/verify/package.json]: Same keywords problem.

Comment (maruilian11) [inline ajax/libs/verify/package.json]: @PeterDaveHello same as?

**Comment by maruilian11** [comment]
@Piicksarn could you help me check this PR ?

File: ajax/libs/verify/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/verify/package.json]: https://github.com/cdnjs/cdnjs/pull/6454#discussion_r47243809

**Comment by PeterDaveHello** [comment]
Update your keywords first please.

**Comment by maruilian11** [comment]
@Piicksarn i have updated the keywords, could you help me check this PR? thank you~

**Comment by Piicksarn** [comment]
@PeterDaveHello, would you please have a look at the pr?
i think it's fine~
Thanks!

**Comment by PeterDaveHello** [comment]
Outdated.

**Comment by maruilian11** [comment]
@Piicksarn i have rebased it could you help me check it again? thank you!

**Pull request merged**